packages/components/inputs/rich-text-utils/src/rich-text-body/dropdown.tsx

```diff
@@ -61,13 +108,13 @@ const Dropdown = (props) => {
                 `}
               >
                 <DropdownContainer>
-                  {props.options.map((item, index) => {
+                  {props.options.map((item: TItem, index: number) => {
                     const itemProps = getItemProps({
                       index,
                       item,
                     });
                     const dropdownItemProps = itemProps;
-                    const isSelected = getIsSelected(props, item);
+                    const isSelected = getIsSelected(props, item as TItem);
```

Comment:
I'm curious about typing the `item` here as we already typed it in the `map` function definition.